### ISSUE TITLE
the Economist, May 16th 2019

### ISSUE THREAD

**Issue by NirViaje** [NirViaje/nirviaje.github.io#59]
## [A new kind of cold war](https://www.economist.com/leaders/2019/05/16/a-new-kind-of-cold-war)

The Economist has issued a special report section covering China and America. There is way too much information for a concise summary. Here are all the links to the sub-topics:

* China and America: Trade can no longer anchor America’s relationship with China
* The view from Washington: In Washington, talk of a China threat cuts across the political divide
* The view from Beijing: In Beijing, views of America have become deeply cynical
* Down on the farm: Why Iowa is Xi Jinping’s favourite corner of America
* Slow boat: Ordinary Americans and Chinese seem to be drifting apart

**Comment by NirViaje** [comment]
## A new kind of cold war
How to manage the growing rivalry between America and a rising China

![20190518_LDD001_0](https://user-images.githubusercontent.com/1320252/58151933-c64aa080-7c9d-11e9-8238-1ea45b240ab7.jpg)

Print edition | Leaders
May 16th 2019
Fighting over trade is not the half of it. The United States and China are contesting every domain, from semiconductors to submarines and from blockbuster films to lunar exploration. The two superpowers used to seek a win-win world. Today winning seems to involve the other lot’s defeat—a collapse that permanently subordinates China to the American order; or a humbled America that retreats from the western Pacific. It is a new kind of cold war that could leave no winners at all.

As our special report in this week’s issue explains, superpower relations have soured. America complains that China is cheating its way to the top by stealing technology, and that by muscling into the South China Sea and bullying democracies like Canada and Sweden it is becoming a threat to global peace. China is caught between the dream of regaining its rightful place in Asia and the fear that tired, jealous America will block its rise because it cannot accept its own decline.

The potential for catastrophe looms. Under the Kaiser, Germany dragged the world into war; America and the Soviet Union flirted with nuclear Armageddon. Even if China and America stop short of conflict, the world will bear the cost as growth slows and problems are left to fester for lack of co-operation.

Both sides need to feel more secure, but also to learn to live together in a low-trust world. Nobody should think that achieving this will be easy or quick.

The temptation is to shut China out, as America successfully shut out the Soviet Union—not just Huawei, which supplies 5g telecoms kit and was this week blocked by a pair of orders, but almost all Chinese technology. Yet, with China, that risks bringing about the very ruin policymakers are seeking to avoid. Global supply chains can be made to bypass China, but only at huge cost. In nominal terms Soviet-American trade in the late 1980s was $2bn a year; trade between America and China is now $2bn a day. In crucial technologies such as chipmaking and 5g, it is hard to say where commerce ends and national security begins. The economies of America’s allies in Asia and Europe depend on trade with China. Only an unambiguous threat could persuade them to cut their links with it.

It would be just as unwise for America to sit back. No law of physics says that quantum computing, artificial intelligence and other technologies must be cracked by scientists who are free to vote. Even if dictatorships tend to be more brittle than democracies, President Xi Jinping has reasserted party control and begun to project Chinese power around the world. Partly because of this, one of the very few beliefs which unite Republicans and Democrats is that America must act against China. But how?

For a start America needs to stop undermining its own strengths and build on them instead. Given that migrants are vital to innovation, the Trump administration’s hurdles to legal immigration are self-defeating. So are its frequent denigration of any science that does not suit its agenda and its attempts to cut science funding (reversed by Congress, fortunately).

Another of those strengths lies in America’s alliances and the institutions and norms it set up after the second world war. Team Trump has rubbished norms instead of buttressing institutions and attacked the European Union and Japan over trade rather than working with them to press China to change. American hard power in Asia reassures its allies, but President Donald Trump tends to ignore how soft power cements alliances, too. Rather than cast doubt on the rule of law at home and bargain over the extradition of a Huawei executive from Canada, he should be pointing to the surveillance state China has erected against the Uighur minority in the western province of Xinjiang.

As well as focusing on its strengths, America needs to shore up its defences. This involves hard power as China arms itself, including in novel domains such as space and cyberspace. But it also means striking a balance between protecting intellectual property and sustaining the flow of ideas, people, capital and goods. When universities and Silicon Valley geeks scoff at national-security restrictions they are being naive or disingenuous. But when defence hawks over-zealously call for shutting out Chinese nationals and investment they forget that American innovation depends on a global network.

America and its allies have broad powers to assess who is buying what. However, the West knows too little about Chinese investors and joint-venture partners and their links to the state. Deeper thought about what industries count as sensitive should suppress the impulse to ban everything.

Dealing with China also means finding ways to create trust. Actions that America intends as defensive may appear to Chinese eyes as aggression that is designed to contain it. If China feels that it must fight back, a naval collision in the South China Sea could escalate. Or war might follow an invasion of Taiwan by an angry, hypernationalist China.

A stronger defence thus needs an agenda that fosters the habit of working together, as America and the ussr talked about arms-reduction while threatening mutually assured destruction. China and America do not have to agree for them to conclude it is in their interest to live within norms. There is no shortage of projects to work on together, including North Korea, rules for space and cyberwar and, if Mr Trump faced up to it, climate change.

Such an agenda demands statesmanship and vision. Just now these are in short supply. Mr Trump sneers at the global good, and his base is tired of America acting as the world’s policeman. China, meanwhile, has a president who wants to harness the dream of national greatness as a way to justify the Communist Party’s total control. He sits at the apex of a system that saw engagement by America’s former president, Barack Obama, as something to exploit. Future leaders may be more open to enlightened collaboration, but there is no guarantee.

Three decades after the fall of the Soviet Union, the unipolar moment is over. In China, America faces a vast rival that confidently aspires to be number one. Business ties and profits, which used to cement the relationship, have become one more matter to fight over. China and America desperately need to create rules to help manage the rapidly evolving era of superpower competition. Just now, both see rules as things to break.

**Comment by NirViaje** [comment]
SPECIAL REPORT: CHINA AND AMERICA
China and America
## Trade can no longer anchor America’s relationship with China
The world should be worried about that, says David Rennie

![20190518_SRP074_0](https://user-images.githubusercontent.com/1320252/58152016-00b43d80-7c9e-11e9-80c6-33afaf4748f9.jpg)

Print edition | Special report
May 16th 2019
Since china emerged from the wreckage of Maoism 40 years ago, the profit motive has become a pillar of stability in its relations with America. Presidential candidates might accuse China of stealing jobs. Spy scandals could simmer. Then corporate bosses and politicians in Beijing and Washington would decide that all sides were making too much money to let relations sour. This focus on mutual self-interest involved queasy compromises. Soon after troops massacred hundreds, possibly thousands, around Tiananmen Square in June 1989, President George H.W. Bush wrote discreetly to Deng Xiaoping to urge joint efforts to prevent “tragic recent events” from harming relations. The financial crash of 2008 revealed a dangerous co-dependency between America the importer of cheap goods and China the thrifty exporter. New terms tried to capture this symbiosis: “Chimerica”, or “the g2”.

Suddenly, however, making money is not enough. In the past couple of years, debate about how to get engagement to work has given way to talk of strategic competition and security threats. Rather than catchy neologisms, scholars are reaching for historical analogies. Some talk of 1914, when clashing British and German ambitions swept aside deep bonds of commerce. China analysts obsess over the “Thucydides trap” that supposedly dooms upstart nations to fighting incumbent powers, as the Greek historian wrote of Sparta and Athens.

China’s rise was always going to cause turbulence. The same country is America’s most daunting strategic rival, its biggest economic challenger and a giant trade partner. That is new. The Japan shock of the 1970s and 1980s triggered demands from politicians for protectionist barriers, as America’s trade deficit in goods with Japan rose 25-fold in a decade. But it was a lopsided political fight: Japan was a dependent military ally. As for the Soviet Union, it was an ideological but not a commercial rival: in 1987 bilateral trade was worth $2bn a year, or less than 0.25% of America’s total trade with the world. In 2018 two-way trade between America and China hit $2bn a day, or 13% of America’s world trade.

Critics argue that elites should have seen this coming. Western leaders had hoped that joining the global economy would make China more like the West, as a growing middle class demanded free speech and more accountable government. They were wrong. The crash of 2008 and spasms of Western populism emboldened Communist Party leaders, notably President Xi Jinping, to reject those norms and assert the party’s supremacy.

America’s shock is made worse by trade in technologies that blur the lines between commerce and national security. The Trump administration’s opposition to letting Huawei, a Chinese technology firm, build 5g telecommunications networks for America or its allies is a taste of that future. Such debates are, at root, about trust, a commodity that mattered less when China exported tennis shoes and televisions rather than microchips that can keep self-driving cars on the road and planes in the air. Yet clumsy forms of self-defence cause harm. Define sensitive technologies too broadly and, in the words of Henry Paulson, a former secretary of the treasury, an “economic iron curtain” may come to divide China and America, choking flows of goods, capital, people and technology, with grave implications for the rest of the world.

![20190518_SRC011](https://user-images.githubusercontent.com/1320252/58152024-0873e200-7c9e-11e9-8000-f7daefd665ea.png)

China’s growing tech prowess is putting new strains on globalisation, beyond old arguments about stolen jobs. The fact that General Motors sells more cars in China than in America used to help both countries manage ideological differences. Today’s supply chains, carrying semiconductors from China to devices in America, actually raise the political stakes.

Million-dollar American weapons rely on microchips sourced from firms around the globe. Critical infrastructure may contain components from a dozen nations, require software updates from a provider on one continent and send streams of real-time data to another. In April a Pentagon advisory board warned defence chiefs to plan for “zero-trust” commercial internet networks. A growing number of business transactions require a lifetime commitment to distant service-providers. In this world, trade relations cannot be quarantined from hard questions about whether countries are partners, rivals or foes.

China has every right to want to grow stronger. Its success in helping hundreds of millions of people to raise themselves from poverty is admirable. It is the relentlessness of its methods that has turned business from a safe space to a field of contention. Western firms worry that before China truly opens up, they will be thrown out—as soon as Chinese firms have learned, bought or stolen enough Western knowhow to become self-reliant.

Nothing to lose but your supply chains
Few Americans have better access to Chinese leaders than Mr Paulson, a longtime proponent of engagement. So it was noticed when in February he declared that, because China has been slow to open its economy since joining the World Trade Organisation in 2001, “the American business community has turned from advocate to sceptic and even opponent of past us policies toward China”. Bosses do not seek a tariff war, he said, but do want a “more confrontational approach”. Businesses are getting that from the Trump administration.

In part, this is explained by the change in occupant of the Oval Office. President Barack Obama also denounced Chinese trade cheating and pressed China to stop stealing commercial secrets. Belatedly, his Pentagon chiefs grew alarmed as China turned disputed reefs in the South China Sea into military outposts. But ultimately Mr Obama put more weight on tackling global challenges, from climate change to pandemics to nuclear proliferation, for which he needed Chinese help. Get-tough policies were endlessly discussed, then often dropped. Mr Trump, by contrast, boasts that solving the world’s problems is not his job.

In part, America has become more confrontational because multinational businesses that oppose barriers to trade have lost clout in a populist age. A new round of export controls for sensitive technologies and still-tougher investment-screening rules loom. That process will not end with a truce in Mr Trump’s trade war.

The American president is as much a symptom as a cause of a change in the way that America thinks about its openness to the world. Voters elected a might-makes-right leader who scorns alliances, who is cynical about the rule of law and universal values and who believes that national interests always come first. Amid espionage fears, visa rules for Chinese students of science and technology have tightened. fbi agents have quizzed scholars visiting from Chinese state-backed think-tanks about government links, and cancelled the visas of some. Rather than China becoming more Western, America is becoming more Chinese.

Meanwhile, officials in Beijing see a sore loser of a superpower, bent on keeping them down. They scoff at the idea that rich, spoiled America really feels threatened, seeing a ploy to extract better terms for American firms to make money. This misses how many people in Washington believe that the China threat is real and matters more than profits or free-market purity. Indeed, officials accuse firms of keeping quiet when Chinese spies steal intellectual property, to preserve face and access to Chinese markets.

A senior American official says that China “emphatically” lied when it promised Mr Obama in 2015 that state-backed actors would stop spying on America for commercial gain. The official laments that, in the frenzied Washington news cycle, few noticed a Department of Justice indictment in December 2018 accusing China’s ministry of state security of ties to a long-standing campaign by the apt10 hacking group, stealing secrets from firms in aviation, space, pharmaceuticals, oil and gas, maritime and other technologies. “They basically got the crown jewels of hundreds and hundreds of the world’s biggest companies,” he says.

The pendulum risks swinging too far. Some sniggered in March when the Committee on Foreign Investment in the United States (cfius), a government agency that screens foreign deals for security concerns, asked a Chinese internet firm to sell Grindr, a gay dating app with 3.3m daily users. Actually cfius may have a point. A gay app could be a blackmailer’s trove, and Chinese police routinely grab data from social media at home. It is harder to take seriously Senate attempts to ban Washington, dc, from using federal money to buy metro trains made in America by a Chinese state-owned company lest on-board security cameras are used for spying.

Although China lacks the formal alliances that made the Soviet Union a global threat, its rise dominates Pentagon debates about the future of war. Since the 1980s America has pursued a “forward presence” doctrine, meaning that its forces were confident about operating close to enemy defences. China’s growing strength confronts Pentagon planners with their hardest decision in years: to find new ways to make combat in the Western Pacific viable, or pull back and force adversaries to fight far from home.

Karl Eikenberry was a China expert in the army who became a lieutenant-general, then ambassador to Afghanistan. Now at Stanford University, he describes commanders grappling with the end of overwhelming American superiority: “There is an intense debate within the American armed forces about how to counter the pla’s accelerating efforts to control the South China Sea.”

This report will look at clashing views in Washington and Beijing about how to manage the technological, military, economic and political aspects of a great-power contest so new that the two sides do not even agree on what successful relations might look like. Rules must be found. Mr Eikenberry’s summary of the military challenge applies to the whole: “A new doctrine is required.”

This article appeared in the Special report section of the print edition under the headline "A new kind of cold war"

**Comment by NirViaje** [comment]
SPECIAL REPORT: CHINA AND AMERICA
The view from Washington
## In Washington, talk of a China threat cuts across the political divide
Amid accusations of theft and espionage, opinions have hardened

![20190518_srp076](https://user-images.githubusercontent.com/1320252/58152170-66082e80-7c9e-11e9-8c9a-a697dfd26fa3.jpg)

Print edition | Special report
May 16th 2019
Last october bosses from some big, innovative companies were invited to an annexe of the White House. Amid the high-ceilinged pomp of the Indian Treaty Room, the executives signed one-day non-disclosure agreements allowing them to see classified material. Then the Director of National Intelligence, Dan Coats, and two senators told them how China steals their secrets.

The unpublicised event was the idea of Senator Mark Warner of Virginia, the senior Democrat on the Senate Intelligence Committee and himself a successful technology investor. He was joined by Senator Marco Rubio of Florida, a Republican on the committee.

Recent arrests of alleged Chinese spies reveal only a small fraction of what is afoot, Mr Rubio says. China “is the most comprehensive threat to our country that it has ever faced”. The aim, he insists, is not to hold China down but to preserve peace. He sees an imbalance in relations between America and China that, if left unaddressed, “will inevitably lead to very dangerous conflict”.

Speaking with rapid precision in his Senate office, Mr Rubio criticises an economic model that presses chief executives to maximise short-term profits. China has learned to use that system to turn firms into “advocates”, he charges. Too often politicians would vow to get tough on Chinese cheating. “Then these ceos would be deputised by China to march down to the White House.”

Venture capitalists have also been invited to Warner-Rubio China road shows. Mr Rubio grumbles that the business plan of some Silicon Valley tech firms is to get bought up, without necessarily caring if the investors are Chinese.

Members of Congress have drafted proposals for a series of new export controls on products deemed important to national and economic security, notably from industries named as priorities in the “Made in China 2025” plan. That is a Chinese map for building world-beating companies in ten high-tech fields. Chinese investments face ever-tighter scrutiny by the Committee on Foreign Investment in the United States (cfius). The Foreign Investment Risk Review Modernisation Act recently extended the remit of cfius to new areas, such as property purchases near sensitive sites. A pilot scheme mandates reviews of foreign stakes in a wide array of “critical technologies”. Mr Rubio names telecommunications, quantum computing, artificial intelligence and any industry that collects large data sets as ones he wants closed to China.

The staging of that October road show—a bipartisan endeavour involving Congress and the intelligence agencies, close to the White House but not inside it—is revealing. Views on China have hardened across official Washington. A tough new consensus unites what might be called America’s foreign-policy machine, including members of both parties in Congress, the State Department, Pentagon, Department of Justice, spy agencies and the president’s own National Security Council. The machine includes the vice-president, Mike Pence, who turned a speech last October into a charge sheet of Chinese misdeeds. Mr Trump stands apart.

Pentagon chiefs and members of Congress are ever more publicly sounding the alarm about China’s intentions towards Taiwan, the democratic island of 24m people that America calls an ally but China claims as its own, saying it must be united with the motherland, by force if necessary. To China’s disquiet, Congress has passed laws signalling solidarity with Taiwan, urging the government to allow cabinet secretaries and American warships to visit the island. Some of President Donald Trump’s closest aides are long-time advocates for Taiwan. As president-elect in 2016 he was persuaded to talk by telephone with the island’s president, Tsai Ing-wen. Since then Mr Trump has blocked proposals for high-profile visits to show support for Taiwan as a democratic ally. He sees allies as a burden, and mighty China as America’s peer.

Whose side are you on?
Discerning a united view of China within Team Trump is hard. Trump aides use harsh language about the country. Referring to repression of Uighur Muslims in the north-western region of Xinjiang, the Secretary of State, Mike Pompeo, called China “one of the worst human-rights countries that we’ve seen since the 1930s”. That tone is a sign of their boss’s willingness to trample diplomatic niceties. But while Mr Trump’s views on China overlap with the Washington machine’s, they are not identical. Many officials are sincerely disgusted by Xinjiang, where perhaps a million Uighurs are being held in “re-education camps”. Asked how business ties between America and China may co-exist with get-tough policies, a senior administration official replies: “Concentration camps do spoil the mood, don’t they?”

Yet cold-war-style discussions of human rights are of little interest to Mr Trump. Michael Pillsbury is a China specialist at the Hudson Institute, a think-tank, and an outside adviser to the White House. In his view, “the president is not a super-hawk on China”. Such issues as Taiwan or Xinjiang do not resonate with Mr Trump as much as trade does, he admits. Even on trade, Mr Pillsbury calls him more cautious than advisers such as Peter Navarro, who would like American firms to leave China. Mr Trump has often said he does not want to hurt China’s economy, notes Mr Pillsbury. “He sees China as a source of profit and investment.”

The machine wants to change the fundamental principles guiding China’s rise. In contrast Mr Trump praises President Xi Jinping for putting China’s interests first.

Yet Mr Trump can be riled by aides telling him that China is “stealing our secrets”. He also sees political risks in any trade deal that can be branded a climb-down. “The president understands very clearly that the Democrats are waiting for him to be soft on China,” says Mr Pillsbury. Senator Chris Coons, a Democrat, agrees that being a hawk on China in today’s Congress is “comparable to the 1950s when there was no downside, politically, to being anti-Soviet”.

Tellingly Mr Trump’s China tariff escalation on May 10th was accompanied by defensive tweets asserting that China yearns for a “very weak” Democrat to win the 2020 election instead. A senior Trump administration official endeavours to reconcile the different camps. The aim is not economic decoupling, he says. But in sensitive industries, “the political and financial risk associated with doing business in China will continue to rise”.

Modern-day Chinese mandarins obsess over differences within the Trump administration, not realising that the hardening of the Washington mood predates and will outlast Mr Trump. Evan Medeiros of Georgetown University, a former principal Asia adviser to President Barack Obama, notes that “the bureaucracy of a much more competitive relationship” is being put in place.

Taking a proper gander
Last November the Department of Justice established a China Threat Initiative, staffed by prosecutors and fbi investigators, to detect Chinese attempts to steal trade secrets and influence opinion, in particular on university campuses. At the Department of Homeland Security, a new National Risk Management Centre watches for high-risk firms working on critical infrastructure. A State Department office formerly focused on terrorism, the Global Engagement Centre, has a new mission countering propaganda from China, Russia and Iran.

Pentagon anxieties about China coincide with a realisation that when troops rely on high-tech kit, cyber-attacks can kill. Mr Eikenberry, the former general, observes that in the 1970s or 1980s perhaps 70% of the technology that mattered to military commanders was proprietary to the government, and the rest off-the-shelf and commercial. “Now it is 70% off-the-shelf, much of it coming from Silicon Valley,” he says. Thus when American trade negotiators debate China policy, “the security people are in the room.”

A study commissioned by the Pentagon, “Deliver Uncompromised”, warns that insecure supply chains place America’s armed forces at “grave risk” from hacking and high-tech sabotage, for instance by the insertion of malware or components designed to fail in combat. The study, by Mitre, a research outfit, notes that modern fighter jets may rely on 10m lines of software code, so it matters if tech firms use code of unknown provenance, as some do.

Pentagon chiefs have created a new Office of Commercial and Economic Analysis whose mission includes scouring defence contracts for Chinese companies, down to third-tier suppliers. James Mulvenon, an expert on Chinese cyber-security, explains that “the Pentagon has decided that semiconductors is the hill that they are willing to die on. Semiconductors is the last industry in which the us is ahead, and it is the one on which everything else is built.” He already sees more high-value defence contracts going to semiconductor foundries in America.

Randall Schriver is assistant secretary of defence for Indo-Pacific Security Affairs and a China specialist. Asked if the Pentagon will press businesses to leave China, he replies carefully. “Companies can do what companies do. We are much more aware of and keen to address vulnerabilities in our defence supply chain.”

Official Washington has moved beyond asking whether China is a partner or a rival. The only debate concerns the magnitude of China’s ambitions. According to Mr Rubio, Mr Xi thinks that “China’s rightful place is as the world’s most powerful country.”

Some political appointees in Mr Pompeo’s State Department sound eager to declare that an East-West clash of civilisations is under way. On April 29th the State Department’s director of policy planning, Kiron Skinner, told a forum hosted by New America, a Washington think-tank, that there was a need for a China strategy equivalent to George Kennan’s containment strategy for the Soviet Union. Not content with that bombshell, Ms Skinner ventured that China is a harder problem. “The Soviet Union and that competition, in a way it was a fight within the Western family,” she said, citing the Western roots of Karl Marx’s ideas. “It’s the first time that we will have a great-power competitor that is not Caucasian.”

Leaving aside the ahistoricism of Ms Skinner’s comments—for China’s Communists drew deeply on Marx and Lenin—they are self-defeating. A clash of civilisations leaves no room for Chinese liberals, let alone for Taiwan, a democracy with deep roots in Chinese culture. As for the idea of containing one of the world’s two largest economies, that would be a nonsense even if American allies and other countries were willing to help, which they are not.

There are more cautious voices. A recent essay for the Paulson Institute by Evan Feigenbaum, an Asia hand in the administration of President George W. Bush, argues that those accusing China of remaking the global order are both misstating and understating the challenge. China is selectively revisionist, wrote Mr Feigenbaum. Rather than seeking to replace today’s international system, it upholds many of the “forms” of multilateralism while undermining “norms” from within the un and other bodies.

In a break between votes, in a windowless office deep in the Capitol, Mr Coons urges Congress to try the hard work of dealing with China as it is and not as America wishes it to be. He does not think China is hostile to the idea of a rules-based order, but concedes that it has “behaved exceptionally badly on the world economic stage”. In today’s Washington, that is dovish talk.

> This article appeared in the Special report section of the print edition under the headline "I spy"

**Comment by NirViaje** [comment]
SPECIAL REPORT: CHINA AND AMERICA
The view from Beijing

## In Beijing, views of America have become deeply cynical
Many officials are frustrated with Donald Trump

![20190518_SRP063_0](https://user-images.githubusercontent.com/1320252/58152339-e75fc100-7c9e-11e9-84eb-9650e61b72ed.jpg)

Print edition | Special report
May 16th 2019
Spend enough time with Chinese scholars and officials who study America, and comparisons will at some point be drawn between China’s relations with America and a bad marriage. It is a revealing analogy. China has interests in other continents, but America is an obsession. Marriage metaphors capture the lingering admiration mixed with envy and resentment that China’s elite harbours for its global rival. In the Trump era, however, a dangerous new emotion is increasingly surfacing: contempt.

Powerful Chinese officials have few incentives to talk to outsiders. But some cadres and scholars known to brief government and party bosses do speak off the record. Leaders are selectively candid with foreign counterparts, and maintain ties to retired Western grandees. It can be said with confidence that China’s ruling classes claim to be deeply frustrated by the America that elected President Donald Trump. It is called a sore loser and a dangerous spoiler, not only unwilling to play a leading role in the world but livid if China becomes more active.

According to this line of thought it is, for instance, maddening to hear America complain about China’s ambitions for the Belt and Road Initiative, President Xi Jinping’s globe-spanning infrastructure plan, when America is no longer prepared to play a leading part in setting global standards, and is too self-centred to invest in connecting the world.

If this were a marital row, one line that sums up the mood in Beijing would be an angry challenge directed at America: “Why do you always think this is about you?” China did not set out to overtake America, it is argued. If it becomes the world’s largest economy, that is because it has a lot of people and wants to give them better lives. Yes, it has enjoyed a successful 40 years, but only thanks to its people’s ceaseless hard work. Still, many regions have been left behind and are crying out for development.

That need to maintain economic growth is a reason for China to fear a trade war with Mr Trump. But it is also a reason for indignation at what is called an American policy of containment. Chinese sources describe a suffocating sense that—just as moderate prosperity comes within reach—a declining America now questions China’s right to achieve that wealth, whether by building strong armed forces or developing advanced technology.

For it is not enough to be rich, they argue. Countries must also be strong, militarily and technologically. The humbling of imperial China by smaller European powers proves that. Often that is the cue for such Chinese sources to bring up the arrest in Canada of Meng Wanzhou, a senior executive with Huawei and daughter of the telecommunications giant’s founder. Depressingly, the argument that follows is a cynical one about the relative strengths of China, America and Canada. It is rarely about the details of the legal case against Ms Meng, arrested at the request of American prosecutors, who accuse her of sidestepping sanctions against Iran. Ms Meng’s arrest is seen as a signal that America might tolerate a richer China but does not want it to be strong. Pro-China protests have ensued in Canada and Hong Kong (pictured).

Discussions of territorial disputes follow similar lines. The Chinese government’s public version of history asserts its sovereignty over reefs and islands in the South China Sea. But in private, when officials protest about America’s insistence on sending warships and planes through those disputed waters, the complaint is that America is showing disrespect, and would never tolerate such impertinence in its own backyard. After 40 years of growing richer, it is time for China to tackle such long-ignored issues, they say.

Other official voices argue that China has not changed its behaviour; it has merely grown larger and more successful. If China is such an abuser of the international order, they ask, why have America and Europe never complained before? In part, Chinese puzzlement is disingenuous. China has repeatedly promised to open markets and grant more equal treatment to foreign companies. After the 20th year of broken promises, patience vanishes. In part, though, those Chinese sources have a point.

Foreign businesses have spent years sending Chinese leaders mixed messages, notes a Beijing-based American. In meetings with central-government leaders, Western bosses would talk up their positive experiences, both out of caution and because they saw little point in raising problems caused by powerful provincial and local barons, knowing that the central government might simply ask those barons to investigate themselves. Having stayed mum in China, American businesses would grumble to their own government, which would take their complaints to the Chinese. But the Chinese would not believe them, thinking they had heard the truth from businesses on the ground.

Mr Trump divides Chinese officials and scholars. An older generation, notably those who were among the first to study in America, is broken-hearted to find him popular with so many voters. Another generation of high-flying, middle-aged officials is more inclined to gloat. Their formative memories of America involve the disasters of the Iraq invasion of 2003 and the financial crisis of 2008. Mr Trump’s diagnosis of America’s ills is quite correct, they sniff. By that they mean he is right to say America should pull troops back from the Middle East and Asia and instead focus on nation-building at home. At the same time, they add, the low quality of his appointed officials points up the superiority of China’s meritocratic one-party system.

The most candid voices admit that China got Mr Trump wrong, at first thinking him a pragmatic New York businessman in the mould of others they have known. China also underestimated the durability of his support. Mr Trump’s escalation of the trade fight shocked Chinese leaders, who assured visiting Western leaders in spring 2018 that his bluster was theatre, and that both sides had too much to lose for a real trade war to start.

Following Mr Trump’s threats to rain “fire and fury” on North Korea in 2017, which genuinely alarmed China, the government in Beijing prefers his current policy. For now that combines indifference to North Korea’s human-rights abuses with a willingness to suspend American military exercises in South Korea as long as the north halts tests of bombs or missiles that threaten America. That is essentially the “freeze-for-freeze” policy that Chinese leaders urged on previous American administrations. Those administrations rejected it as a betrayal of Asian security alliances.

Liberals who love Trump
Throughout 2018 foreign politicians and business leaders visiting Beijing were struck by an unexpected phenomenon which might be termed “Liberals for Trump”. This involved reformist Chinese scholars discreetly welcoming Mr Trump’s pugnacious ways. They saw outside pressure as the best way to force through needed changes, from the dismantling of state-run monopolies to the opening of markets. Those liberals are warier now—China feels under attack and Mr Trump seems less keen on structural reform.

Chinese reformers never exactly admired Mr Trump. It is more that they hoped America’s president was a bigger bully than Mr Xi. Despite Mr Xi’s swing back to more authoritarian rule, he has plenty of critics in elite Beijing. Some call him a statist who does not understand economics. Others blame his assertive rhetoric about China’s rise for a backlash abroad, and say he has bungled the American trade war. The puzzle is to know whether such Beijing grumblers matter more or less than the American establishment grandees who deplore Mr Trump at dinner parties in Washington, dc. Others who are angry with Mr Xi are no friends of the West. Bonnie Glaser, a well-connected China specialist at the Centre for Strategic and International Studies, a Washington think-tank, recently reported that the upper ranks of the People’s Liberation Army are gripped by fear, amid a sense that an anti-corruption drive ordered by Mr Xi has gone too far.

Mr Xi has told foreign visitors that he is exasperated by American inconstancy. According to a leaked diplomatic memo, Mr Xi complained to eu leaders at a summit last July that America had walked away from the World Trade Organisation just when China had at last managed to join it. He also pointed out that Mr Obama had persuaded him to join the Paris accord on climate change, only for Mr Trump to pull America out.

Impatience between the two giants is nothing new. Still, it matters that China’s faith in America’s future is waning. Chinese officials used to want America’s respect, asking why the superpower could not accept that their political system is a good fit for China. Chinese officials still go out of their way to note that America is a much stronger and richer country whose enmity they do not seek. But America’s good opinion of China matters less to them.

It would be especially bad if China’s ruling classes began to believe the charge that they have levelled for years: that America is bent on containing China. If a vengeful America wants to hurt China, there are few incentives for Chinese officials to propose imaginative concessions or urge reforms that might repair ties with America. In geopolitics as in marriage, contempt is an emotion that leads to bad outcomes.

> This article appeared in the Special report section of the print edition under the headline "Same bed, different dreams"

**Comment by NirViaje** [comment]
SPECIAL REPORT: CHINA AND AMERICA
Down on the farm
## Why Iowa is Xi Jinping’s favourite corner of America
But even here, attitudes towards China are changing

![20190518_SRP059_0](https://user-images.githubusercontent.com/1320252/58152674-b16f0c80-7c9f-11e9-8277-3b364a3a1102.jpg)

Print edition | Special report
May 16th 2019
When rick kimberley showed Xi Jinping around his farm in Iowa in 2012, he explained how modern seeds, big machines and computers had doubled crop yields since he began farming in 1972. His Chinese guest, who was then vice-president and months away from assuming the leadership of the Communist Party, pronounced the farm a model to study. When he speaks, China acts.

A replica of Mr Kimberley’s property is being built in Hebei province, northeast of Beijing, as a demonstration farm. He is now honorary dean of the Kimberley Agricultural Business School in Shaanxi province. Thousands of Chinese visitors have trekked to his farm in Iowa, many eager to be photographed on the John Deere tractor on which their leader sat. Mr Kimberley would have traded it in by now, but Chinese firms have asked about shipping it to the motherland.

Mr Xi’s first visit to Iowa was in 1985, as leader of a five-man agricultural delegation. His business cards said he was the director of an animal-feed association. His hosts took him to farms and feed mills. He ate roast hog and went on a cruise on the Mississippi River. Iowa’s then governor, Terry Branstad, received the Chinese guests. This Iowan kindness was a lucky investment. Unbeknown to his hosts, young Mr Xi (pictured) was party secretary of a county in Hebei and son of a member of China’s politburo, Xi Zhongxun, who had visited Iowa in 1980.

Today, Mr Branstad is America’s ambassador to China, and delights in talking up his long friendship with Mr Xi. China’s president seems attached to those memories, too. During his second visit in 2012 he spent an hour with Iowans who had hosted him in 1985. “For me, you are America,” Mr Xi enthused.

China has duly showered Iowa with demonstrations of amity. For the past four years orchestras on American concert tours have given free concerts in Muscatine, the town of 24,000 where, in 1985, Mr Xi stayed with a local family, the Dvorchaks. Grateful Muscatine high-school students have enjoyed free study tours of China, funded by Wanxiang, a Chinese maker of car parts.

Gary Dvorchak was at college when China’s future ruler borrowed his teenage bedroom, complete with Star Wars figures on the shelves. His reward came in 2015 when his family was invited to Beijing to dine with President Xi, his wife and his daughter. Today Mr Dvorchak is a business consultant in Beijing. In the face of continued unequal treatment for foreign firms, the impatience of American businesses is at “boiling point”, he laments. Often asked to meet delegations of Iowan farmers and entrepreneurs, he tries to warn them about cultural differences. Chinese business partners learn to trust slowly. Americans are in a hurry, and trust strangers until given cause not to—but once disappointed will walk away.

From the new world
Mr Dvorchak’s former home in Muscatine is now the Sino-us Friendship House, a museum displaying photographs of Mr Xi in Iowa. Its developer, Cheng Lijun, owns several properties in Iowa and is bringing up his children there. He thinks that ordinary Americans still welcome Chinese investment. But Chinese businesses feel “a lot of invisible pressure” from America’s government, which sees a spy scandal in every bid for a business that uses technology, he sighs.

In 2017 America sold China soyabeans worth $12.4bn, many from Iowa. Then Mr Trump launched his trade war and China slapped tariffs on American soyabeans. Tim Maxwell grows them near Muscatine. He backs the president even if his sales are hit: “We’re going to feel a bit of a sting for a couple of years, because he is not going to let anyone push him around, and I’m all for that.”

Sarah Lande (pictured) helped to organise Mr Xi’s visit in 1985, giving him a lift in her red convertible (she regrets declining his request to drive it). She hosted him again in 2012. But this pioneer of engagement senses a new wariness among her neighbours. “People are influenced by what they read in the papers, that China is spying on us,” she says. If a young Chinese official were to visit today, she is not sure his delegation would get the same sort of welcome: “What we show them might be a bit broad-brush now.”

> This article appeared in the Special report section of the print edition under the headline "Down on the farm"

**Comment by NirViaje** [comment]
SPECIAL REPORT: CHINA AND AMERICA
Slow boat
## Ordinary Americans and Chinese seem to be drifting apart
Cultural and educational exchanges are under strain

![20190518_SRP075_0](https://user-images.githubusercontent.com/1320252/58152732-d794ac80-7c9f-11e9-8330-203db85a0a52.jpg)

Print edition | Special report
May 16th 2019
During a state visit to Beijing in November 2017 President Donald Trump invited his Chinese counterpart, Xi Jinping, to watch a video of his young granddaughter, Arabella, singing and reciting poetry in Mandarin. The daughter of Ivanka Trump and Jared Kushner learned her language skills from a nanny and at a private school. Many parents think that having studied Chinese looks good on a college application alongside ballet or violin lessons, says Scott McGinnis, a professor of Chinese at the Defence Language Institute, speaking in a personal capacity.

A survey in 2017 by American Councils for International Education estimated that 227,000 school-age Americans regularly wrestle with Chinese tones and radicals. At roughly the same time 363,000 students from China were hosted by American colleges and schools. Those numbers make talk of a cold war seems outlandish. Beyond trade, people-to-people exchanges look like sturdy guardrails to keep relations on track. From Deng Xiaoping on, the past four Chinese leaders all sent a child to study in America (Mr Xi’s daughter was at Harvard). Soviet leaders did not enroll their young in Ivy League institutions.

Alas, those guardrails are weaker than they may appear. In many places, Chinese student numbers are dropping. The University of Iowa, for example, had seen Chinese enrolments rise five-fold between 2007 and 2015, with the effects still visible in the streets of Iowa City, where bubble-tea outlets and noodle bars cater to thousands of Chinese students.

But numbers peaked in 2015 and have since fallen by about 39%. Sydney Ji, a graduate student from Shanghai, says that it has become harder for Chinese students to secure or renew visas (rules are especially tight for students in some high-tech fields). One of her own friends is returning home after failing to secure the right to stay and work. The American embassy in Beijing has begun issuing leaflets to Chinese students who are granted American visas urging them to “learn with an open mind”, and enjoy the free thinking and debate of college life.

Visa rules are likely to get stricter still. In April Christopher Wray, the director of the fbi, urged academic institutions to be more mindful of how others may exploit America’s “open, collaborative research environment”, accusing China of sending graduate students and researchers, among others, to steal innovations. The fbi chief also expressed concern about Confucius Institutes (cis), Chinese-funded outposts based in American universities, trying to win hearts and minds with Mandarin lessons and cultural events.

In 2018 Senator Marco Rubio, a Republican, urged universities in Florida to consider closing cis on their campuses. Across America, at least ten have shut in the past year, leaving about 100. China’s influence on American campuses, via student associations with close links to diplomatic missions, is a frequent topic in congressional hearings. The State Department recently ceased funding a network of American Cultural Centres at Chinese universities in the face of official harassment—including an episode in which Terry Branstad, America’s ambassador, was denied access to one funded by his own embassy.

Though demand for language teaching for school-age children remains strong, the number of college-age students choosing Chinese fell to 53,000 in 2016, a 13% drop since 2013. That is telling because older students need to think that they may want to live or work in a country, says Mr McGinnis. China is becoming less attractive, he concludes with some sadness. The number of Americans studying in China peaked in the 2011-12 school year at nearly 15,000 and was down to just under 12,000 in 2016-17.

Behind all such statistics lie real people. When asked about Chinese students who may feel under suspicion, Mr Rubio pauses. “That is one I struggle with,” says the senator, himself the son of Cuban immigrants. America cannot ignore China’s use of students to acquire technology, he argues. But students exposed to American freedoms may call for change at home. “I don’t want to trigger xenophobia in which every Chinese student in America is presumed to be a spy until proven otherwise,” he says. In these populist times, others may feel less squeamish.

> This article appeared in the Special report section of the print edition under the headline "Slow boat"

**Comment by NirViaje** [comment]
SPECIAL REPORT: CHINA AND AMERICA
Competing in technology
America still leads in technology, but China is catching up fast
Commercial competition is turning into a zero-sum contest

![20190518_SRP073](https://user-images.githubusercontent.com/1320252/58153080-c8622e80-7ca0-11e9-92fa-4e25db07672e.jpg)

Print edition | Special report
May 16th 2019
Arare thing happened in an industrial park near Washington, dc, last November. Construction began on a $3bn extension to a semiconductor foundry owned by Micron Technologies, a maker of advanced memory chips, based in Idaho. “A few years ago, opening that sort of extension would have people saying, well, that is going to be moving to China soon, isn’t it?”, observes James Mulvenon, an expert on Chinese cyber-policy and espionage.

Not now. Instead, that Micron foundry is a glimpse of the future. Trust in China has collapsed among American government and business bosses, and a consensus has grown that Chinese firms have closed the technological gap with Western rivals with indecent speed and by illicit means.

Today’s tensions make the original cold war look simple. In 2018 China accounted for 57% of Micron’s net sales. In the 1960s and 1970s American tech companies did not rely on Soviet customers. But Micron is a symbol, several times over, of how commercial competition is turning into a zero-sum contest, in which one side wins at the other’s expense. In 2015 Micron rebuffed a $23bn takeover bid from a Chinese state-backed investment fund, saying that it thought such a deal would be blocked on national-security grounds by the Committee on Foreign Investment in the United States (cfius). In 2018 the Department of Justice indicted a state-owned Chinese firm, its Taiwanese partner and three individuals on charges of stealing trade secrets relating to Micron’s memory chips—technology worth tens of billions of dollars. That followed lawsuits and countersuits in which the accused Chinese firm asserted that it owned the relevant patents in China and was therefore Micron’s victim. A Chinese court sided with it, then Micron was hit with an antitrust probe.

China hawks in Washington say the zero-sum game is about broken laws. “Put plainly, China seems determined to steal its way up the economic ladder at our expense,” declared Christopher Wray, the director of the fbi, on April 26th, adding that nearly all the agency’s 56 field offices are working on economic spy cases “that almost invariably lead back to China”. Between March and November 2018, the Department of Justice indicted a dozen individuals and entities it says were directed by the Chinese government to obtain commercial secrets from 15 companies, predominantly in aerospace and high technology.

Others say the zero-sum game involves broken promises to American workers. They recall American political leaders assuring workers that high-value manufacturing would stay in America, even as globalisation carried cheap jobs to China.

Using his chairmanship of the Senate committee on small business and entrepreneurship as a bully pulpit, Senator Marco Rubio of Florida in February issued a report condemning China’s plans to become a global powerhouse in ten high-tech fields, from artificial intelligence (ai) to aviation, as laid out in the “Made in China 2025” (mic2025) plan issued by the State Council in 2015. Should America let China become the global leader in innovation and manufacturing, “this would be an unacceptable outcome for American workers,” Mr Rubio writes in his report.

In a mark of these populist times, Mr Rubio is not afraid to argue that government has a direct role to play in defending blue-collar factory jobs. Manufacturing provides more stable employment than services, the Rubio report avers. It urges America to use industrial policies, including tax changes and export controls, to defend industries from robotics to tractor-making.

It’s a steal
Not all senators are as vocal as Mr Rubio, nor as keen on export controls. But deepening distrust of China is a bipartisan norm in Congress. The views of American businesses in China are a bit more nuanced, as shown by the 2019 business-climate survey of the American Chamber of Commerce there, issued in February. Nearly 70% of firms say they are profitable. Still, there are warning signs. In the AmCham survey, half of all American technology firms say they limit investments in China because of inadequate protection of intellectual property (ip), even after years of Chinese promises to get serious about it.

China has become tougher on acts of piracy, from fake consumer goods to breaches of patents. But foreign executives still tell horror stories about pressure to share secrets with local partners and cyber-attacks on company servers back home. Depressingly, 13% of member firms in the AmCham survey said that their greatest ip risk was theft by their own employees.

There are several ways in which economic competition can become zero-sum, and all can be seen in China today. Theft is just one. Another is the pursuit of import substitution, aiming to replace imports with domestic alternatives, by fair means or foul. America is in a funk about losing its edge, but it is still home to global champions from aerospace and semiconductors to software and self-driving vehicles. Its officials worry that mic2025 commits China to being world-class in all those sectors.

Since 2015 supporting plans and road maps published by government research agencies set out hundreds of market-share targets for Chinese firms, declaring, for instance, that 80% of electric or hybrid “new energy” vehicles sold in China must be domestically produced by 2025. Chinese officials, facing a worldwide backlash, now downplay those targets. Strictly-censored state media have stopped using the term mic2025. But the policy itself has not been repealed. Speeches by party chiefs ring with calls for “self-reliance” and “indigenous innovation”. Other Chinese technology sectors are being encouraged to comply with a policy called “civil-military fusion”, a national strategy backed by top leaders and funding from opaque national-security budgets.

That militarisation of some Chinese technology imposes costs as well as benefits, notes Mr Mulvenon. Those costs include the risks for Western firms of doing work that supports the brutal techno-dystopia that China has built in Xinjiang. “The good news is that the Chinese are going to discover that autarky is hard,” says Mr Mulvenon. Americans have watched China stealing and reverse-engineering one generation of technology, he says, then having to steal the next after failing to master the underlying science. “That model is incredibly inefficient.”

China is willing to spend what it takes, showering would-be champions with billions of dollars in subsidies and prodding local firms to place orders. Among the beneficiaries is the Commercial Aircraft Corporation of China, whose c-919 commercial airliner is intended as a direct competitor to Boeing’s 737. State planners have set a goal of a 10% domestic market share for Chinese airliners by 2025. The c-919 has had teething troubles, making that timetable ambitious. But success for China could quickly feel zero-sum in America, whose top export category to China in 2017 was civilian aircraft, worth $16.3bn. The Rubio report laments that at least ten American firms supply vital parts to the c-919.

![body_4](https://user-images.githubusercontent.com/1320252/58153073-c39d7a80-7ca0-11e9-9dcc-45c4467ef8f5.jpg)

China has created big brands in such fields as electric vehicles and batteries, in part by shutting foreign rivals out. Protectionist barriers have also allowed Chinese internet firms to grow. In 2009 the ten largest internet companies by revenue were American. Now several are Chinese (see chart).

Still, it is a mistake to exaggerate China’s strengths in big-data analysis and ai, according to Dieter Ernst of the East-West Center, a think-tank in Hawaii. A near-total lack of privacy protection may help sweep up lots of data, but American firms are better at advanced algorithms that make ai less dependent on big data sets, Mr Ernst wrote. Big Chinese applications are still mostly powered by American-designed chips, which remain world-beating.

America has other advantages. Joy Dantong Ma of MacroPolo, the in-house think-tank of the Paulson Institute, examined the origins of leading speakers at the most prestigious ai gathering. Most came from American universities and tech firms, she found. Crucially, though, more than half those American stars were foreign-born. Team Trump’s visa clampdown imperils that.

Some forms of competition can be fair but still end with the gains going mostly to one side. Notably, some technological fields give a “first-mover advantage” that offers huge rewards to countries or businesses that take an early lead, allowing them to set standards that later entrants have little choice but to follow. In April the Defence Innovation Board, a Pentagon advisory committee of Silicon Valley luminaries, issued a report warning that China is on track to pull off this feat in the race to dominate 5g mobile telecommunications. This next generation of wireless technology promises to revolutionise existing industries and invent whole new ones with data speeds about 20 times those of 4g.

A decade ago American firms took an early lead in 4g, setting standards for new handsets and applications that spread worldwide. That dominance helped Apple, Google and other American businesses generate billions of dollars in revenues. China learned its lesson, investing $180bn to deploy 5g networks over the next five years and assigning swathes of wireless spectrum to three state providers. In America the same part of the spectrum is largely off-limits commercially because it is used by the federal government. American firms are experimenting with a different part of the spectrum that has some advantages under laboratory conditions but is easily blocked by buildings and trees. For this reason, in spite of American pressure on allies, much of the world is likely to adopt China’s handsets, chips and standards, the Pentagon board concludes. Since America’s armed forces are expected to operate worldwide, they must prepare to send data through a “post-Western” world of wireless technology and through “zero-trust” networks, studded with components from such Chinese firms as Huawei. That will mean more focus on encryption and security.

Home of the splinternet
Some technology contests look more benign. As China and America wall off their respective digital markets from one another, each will look for growth in the rest of the world. A divided world wide web, or “splinternet”, is already a reality, as China’s internet grows behind a great firewall of censorship. American champions like Amazon are promoting payment services in India. China’s Alipay service is active in Brazil. China is exporting surveillance systems and censorship algorithms to police states from Ethiopia to Venezuela. With a change in direction, America could make a virtue of an internet that respects privacy. Western biomedical firms and gene-editing laboratories could make a virtue of stricter ethics.

It is unhelpful that Mr Trump is a techno-curmudgeon. He has proposed budgets that slash scientific-research funds, though Congress reversed them. After two recent crashes of Boeing 737 Max airliners, he tweeted that “airplanes are becoming far too complex to fly”. Still, last year Mr Trump signed a bipartisan bill authorising $1.3bn for quantum-computer research. The aim is to keep ahead of Chinese work on computers that harness the laws of quantum physics to achieve processing speeds out of a science-fiction film. America leads this field, but Xi Jinping deems it a national priority, quizzing scientists who have returned from quantum laboratories in America and Europe. Should China succeed, it could develop almost unhackable satellite communications and quantum radar to detect the stealthiest planes and submarines.

Such a success would turn a technology contest into an arms race. America would then have to decide whether this China can be deterred or whether one day it might use its new capabilities.

China is on track to dominate 5G mobile tele-

communications

China is on track to dominate 5G mobile tele-

communications

> This article appeared in the Special report section of the print edition under the headline "One-party tech"

**Comment by NirViaje** [comment]
SPECIAL REPORT: CHINA AND AMERICA
Military development
## America’s military relationship with China needs rules
Armed forces are so different now that a framework for engagement has not yet caught up

![20190518_srp100](https://user-images.githubusercontent.com/1320252/58153209-0f502400-7ca1-11e9-8249-20b0a3b28cd9.jpg)

Print edition | Special report
May 16th 2019
Beep, beep, beep went the first satellite to orbit Earth, the primitive Sputnik 1, launched in 1957. No matter that it could do little else. That Soviet communists had won the first space race sparked an American crisis of confidence. This had useful effects. Abroad, America strengthened such alliances as nato. At home, vast sums were poured into science. The Sputnik crisis felt like a loss of innocence—the enemy was overhead. But the actual Soviet threat had not changed much. The Soviet Union was, as before, a nuclear-armed foe, bent on spreading a rival ideology.

Now America is having a crisis of confidence about China, and the cause is not one Sputnik moment but many smaller ones in a row. Talk to strategists in America and China—military officers, politicians, business bosses and scholars—and it is shocking how many say the chances of a limited conflict are underestimated.

In part that is because China’s armed forces are catching up fast. America spent 17 years becoming expert at sending drones to find and kill individual terror suspects half a world away. Meanwhile China retired old Soviet weapons and acquired advanced fighter planes and warships. It invested in anti-ship missiles to increase the cost to America of intervention in its near seas, and in fleets of submarines (though its subs are still noisy compared with America’s). It fortified small islands and reefs in contested waters of the South China Sea with missiles, radar domes and runways (pictured). President Xi Jinping urged the navy to develop an ocean-going mindset, now that ties of commerce and security bind China—for millennia an inward-looking, agrarian power—to the sea. China has a lead in hypersonic glide weapons, travelling at a mile a second, against which aircraft carriers currently have no reliable defences. Ask about China’s weaknesses, and American officers will mention rigid chains of command which give little autonomy to junior officers. They wonder, too, whether different services could work together in complex missions such as invading Taiwan, the democratic island that China claims as its own.

All-out war for Taiwan is not the most urgent flashpoint. The latest China Military Power Report, sent annually to Congress by the Pentagon, sees “no indication China is significantly expanding its landing-ship force necessary for an amphibious assault on Taiwan”. Instead, planners fret about efforts to push America out of China’s near seas and beyond the “first-island chain” that includes Japan and Taiwan. American ships and planes regularly exert legal rights to cross the South China Sea, triggering Chinese responses that could escalate unpredictably.

Far out
This era of doubt even has its own emblematic Chinese satellite, the Shijian 17. Officially an experimental craft, testing new propulsion systems and imaging devices for spotting space debris, American scientists and military leaders have watched the sj-17 perform remarkable manoeuvres since its launch in 2016, scooting between three different Chinese satellites high above the Earth and parking itself within a few hundred metres of one of them. China, like America, is becoming skilled in the dark arts of anti-satellite warfare. It first tested a satellite-destroying missile in 2007, strewing debris in space, and is thought to have tested anti-satellite lasers and jammers. Last year Mike Pence, the vice-president, included “highly sophisticated” Chinese satellite manoeuvres as one of the reasons to set up a “Space Force”, a new service branch drawing on a broad range of specialists.

Strategists talk about the difference between capabilities and intentions. Alarm at China is eroding that distinction. When the us-China Economic and Security Review Commission, a congressional oversight panel, held a hearing on China’s space programmes last month, a Pentagon representative, William Roper (the assistant air force secretary for acquisition, technology and logistics), noted that the commission was really asking whether America is in a strategic competition with China in space. “I hope you conclude ‘yes’,” he told them. Noting America’s vast lead in space—it deploys more than half the world’s declared spy satellites—Mr Roper asserted that “countries like China have already demonstrated their intention to escalate hostilities into space.”

![20190518_SRC851](https://user-images.githubusercontent.com/1320252/58153216-124b1480-7ca1-11e9-814b-0d9b5f9f57b4.png)

President Donald Trump takes the idea of a Chinese space challenge seriously, says Michael Pillsbury, an outside adviser to the White House. “The Space Force is all about China.” He expresses dismay at China’s 38 orbital launches in 2018, surpassing America’s 34 (see chart). “That shouldn’t be happening.”

The mood of alarm is bipartisan. A space-threat assessment published in April by the Centre for Strategic and International Studies, a think-tank in Washington, dc, opens with a warning from Jim Cooper of Tennessee, a Democrat who chairs the House subcommittee that oversees the space programme: “The risk of a space Pearl Harbour is growing every day…Without our satellites we would have a hard time regrouping and fighting back. We may not even know who had attacked us, only that we were deaf, dumb, blind and impotent.”

Chinese experts suspect unseemly panic. After all, America tested its first anti-satellite weapon in 1959, and most of China’s space feats, from manned flight to the creation of a network of navigation satellites, were pulled off by America decades ago.

A leading maritime strategist, Hu Bo of Peking University, complains that Americans have the bad habit of treating China’s intentions and capabilities as one and the same, perhaps because they consider Chinese power “inherently evil”. As soon as China has a missile with the range to hit the island of Guam, America charges that China is “threatening Guam”, he adds. By the same logic Beijing is in peril, as it lies within range of American bombers and missiles. “But China doesn’t go around claiming that the United States is threatening Beijing.” Mr Hu sees an America that had grown used to feeling invulnerable.

A common complaint in Chinese national-security circles is that America’s mood has turned very suddenly, even though China’s core interests, from its territorial claims over Taiwan to preserving its one-party system, have not changed in decades.

America has changed, a lot. The National Security Strategy (nss) of 2006 declared that America “seeks to encourage China to make the right strategic choices for its people, while we hedge against other possibilities.” The nss of 2017 calls engagement mostly a failure, and charges: “China seeks to displace the United States in the Indo-Pacific region, expand the reaches of its state-driven economic model and reorder the region in its favour.”

Chinese security experts assume that the explanation is simple, and lies in China’s growing military and economic “hard power”, says Zhao Tong of the Carnegie–Tsinghua Centre for Global Policy, a Beijing-based think-tank. He sees a flaw in that argument, however. America’s mood change was swift, China’s rise gradual. Mr Zhao has a somewhat different explanation. It is not just that China is stronger, but that it has become more willing to show off that strength—an assertiveness connected to a renewed emphasis on ideology in Chinese domestic politics. That made the world realise that China is not about to embrace anything resembling Western values, Mr Zhao suggests.

Other misunderstandings lurk. When smaller neighbours complain that China is threatening them, Chinese security folk are convinced that America must have put the tiddlers up to it. The way they tell it, when China acts tough it is in self-defence, showing that it cannot be pushed around. “I am very anxious, because China has not acquired the capacity to look at issues from the perspective of others,” says Mr Zhao.

International-relations scholars call the most lethal forms of misunderstanding a “security dilemma”. It can arise when one state takes defensive actions which are mistaken for acts of aggression by another, making all sides less safe. America and China risk such dilemmas today, especially in novel fields of competition.

If strategists spend time counting anti-ship missiles and studying China’s new marine-combat units, they also spend much time thinking about assets and weapons that cannot be seen and for which no rules of war exist, from cyber-weapons to compromised supply chains. Nowhere is this truer than in cyber-warfare, a field so shadowy that China and America do not even agree on basic definitions, such as what constitutes an unacceptable act. Some sound almost nostalgic for the grim but familiar doctrines of the East-West nuclear stand-off during the original cold war.

Thinking about the unthinkable
Away from the din of daily headlines about trade wars and tariff fights, discreet efforts are under way to see if America and China can agree on some basic norms and principles to avoid disastrous clashes or miscalculations in the cyber-domain. These efforts explicitly take historic nuclear arms talks as a model, reviving such half-forgotten cold-war phrases as “confidence-building measures” and “no first use” pledges.

American and Chinese think-tanks have held quiet meetings to talk about actions so disastrous that both countries might be willing to forswear them. The Carnegie Endowment, based in Washington, dc, has suggested a ban on attacks against command-and-control systems governing nuclear forces, and “extreme restraint” over undermining trust in flows of financial data vital to global stability. A group of government experts convened by the un has proposed a norm against attacking critical infrastructure, like dams or power grids. Big technology companies and business leaders have begun debating lists of actions that could become as taboo as mustard gas or anthrax in the physical world, such as testing cyber-weapons “in the wild”, in computer networks connected to the outside world.

Trust is proving a stumbling block to cold-war-style treaties to outlaw such tools. Unlike nuclear warheads, cyber-weapons cannot be counted, and their destruction can never be verified. To date, Chinese experts and officials have proved reluctant to talk about how China’s cyber-warriors operate. America disagrees with China about which forms of cyber-espionage, though annoying for rivals, are to be expected. America draws the line at government spying that steals trade secrets and hands them to favoured companies. China promised to stop such spying in a 2015 agreement between President Xi Jinping and Barack Obama, but American officials insist that the pledge has been broken, with China merely trying harder not to get caught by putting operations under its main spy service, the ministry of state security.

Navigating this new world of known and unknown attacks may require both sides to make painful concessions. In October 2018 a retired colonel from the People’s Liberation Army, Lyu Jinghua, and a former Israeli atomic-energy official, Ariel Levite, published a proposal for a grand cyber-bargain in China Military Science, a pla-sponsored academic journal. The paper suggests that America recognise China’s right to police and censor its own internet aggressively, dropping any insistence that the internet should be a place of free speech and inquiry worldwide. In return, it proposes that China’s cyber-police use their formidable powers to prevent and punish cyber-attacks launched from Chinese territory.

Such proposals lack the drama of America’s response to the Sputnik shock, a space race that put man on the moon and spurred inventions vital to modern life. But the Sino-American confrontation must be managed. Arguably, it already amounts to an undeclared cyber-war. Both sides have overriding interests that can be listed and compared. Maybe one day those lists will become a treaty, making the world safer. Alas, that day has not come yet.

> This article appeared in the Special report section of the print edition under the headline "Sputnik moments"

**Comment by NirViaje** [comment]
SPECIAL REPORT: CHINA AND AMERICA
Trade
## The trouble with putting tariffs on Chinese goods
They rarely work as intended

![20190518_srp070](https://user-images.githubusercontent.com/1320252/58153294-41618600-7ca1-11e9-8c96-d6dd5e24e817.jpg)

Print edition | Special report
May 16th 2019
Donald trump is not the first American president to promise a tougher line on China, but he is the first to make a trade war sound like a rent renegotiation. “I am a Tariff Man,” he tweeted last December, boasting that America is “taking in $billions” thanks to tariffs he has imposed (never mind that tariffs are a tax, mostly paid by American consumers). Mr Trump makes America’s markets sound like a valuable piece of real estate which foreigners should pay more to access. Or as he puts it: “When people or countries come in to raid the great wealth of our Nation, I want them to pay for the privilege of doing so.”

As China grew, politicians typically accused it of not “playing by the same rules”. Mr Trump is different. He is not very fussed about rules. He says that he does not blame China for putting its interests first and for stealing American jobs. He blames his predecessors who allowed that theft to take place.

When China’s business and policy elite ponders the trade war, it is not uncommon to hear Mr Trump described as a pragmatic businessman under the control of a cabal of crazed economic nationalists. In fact, trade is one of the few policy issues on which Mr Trump came into office with fixed beliefs, forged in the 1980s at a time of trade tensions with Japan and Germany. In contrast, his inner circle has spent a lot of time squabbling over trade policy, occasionally in full hearing of stunned Chinese negotiators. Officials in China are slightly obsessed with the president’s chief trade adviser, Peter Navarro, an abrasive academic who would like to decouple the Chinese and American economies. In truth, Mr Navarro’s influence is limited. His main strength is that he represents the world view of trade-union Democrats whose votes Mr Trump needs to be re-elected.

The United States Trade Representative, Robert Lighthizer, was raised in a rustbelt railway town and sees fighting to protect manufacturing workers as the proper work of government. He cut his teeth negotiating with Japan for the Reagan administration. What unites this odd bunch is a shared narrative: that China schemed and cheated its way to stealing American jobs and that those jobs could be dragged home by using enough force, just as it happened with Japan two generations ago.

Back then Japan and Germany placated America by agreeing to strengthen the yen and the d-mark against the dollar, making American goods a bit more competitive. Japan was bullied into voluntarily restricting exports of everything from textiles to cars. More constructively, Japanese firms opened car factories in America, bringing Japanese quality management with them.

Alas for the odd bunch, the solutions imposed on Japan are inapplicable to China, and history will not repeat itself. For one thing China is not about to let its currency strengthen by 50% or more against the dollar. For another, Chinese carmakers or telecommunications giants like Huawei are not very welcome to invest in America, where they stand accused of stealing technology and threatening national security.

Team Trump’s narrative also refuses to acknowledge the logic of global supply chains. The popular history of how American jobs migrated to China overplays the cunning of Chinese officials and underplays the role of multinational companies from Asia and beyond. In many low-end manufacturing industries, the forces of globalisation sent jobs to China when it offered low wages, cheap land and tax breaks. Foreign firms trained Chinese managers to run export-quality plants.

Now, as Chinese wages are rising and Mr Trump’s tariffs are creating unmanageable political risks, manufacturing jobs are leaving after a 30-year sojourn, heading for South-East Asia and beyond. Getting history right matters because Mr Trump’s trade rhetoric is so steeped in nostalgia. Douglas Paal, who held top Asia posts in the Reagan and first Bush White Houses, sees a defect in every fight based on trade law: “The structure doesn’t allow for the voices of the industries of the future.”

Sometimes a single industry’s fate sums up an era. In the 1970s American factories produced over 15m bicycles a year. Today over 95% of bikes sold in America are imported, overwhelmingly from China. They use decades-old technology, but the Trump administration wielded special “section 301” powers, meant to safeguard the most precious intellectual property, to slap a 10% tariff on Chinese bicycles last September, raised to 25% on May 10th.

I want to ride it where I like
For anyone seeking evidence that trade wars are good for American workers, the bicycle aisle of the Walmart Supercentre in Moline, Illinois, looks promising. Alongside Chinese-made cycles from brands like Huffy or Kent, the racks hold stirringly patriotic machines: mountain bikes carrying the shield-shaped logo of the Bicycle Corporation of America (bca) and tags in the colours of the American flag, bearing the slogan “Bringing Jobs Back to America!” and giving a factory address in South Carolina.

![20190518_SRC009](https://user-images.githubusercontent.com/1320252/58153321-4fafa200-7ca1-11e9-99a9-56aed5114adc.png)

That Walmart aisle is misleading. Arnold Kamler is chief executive of Kent International, a family firm based in New Jersey that sells about 3m bicycles a year to Walmart, Target and other shops. He remembers how, in the late 1980s, Chinese-made bikes sold in America at prices that made no sense and then kept falling by a further 5-10% each year. Kent closed its New Jersey plant in 1991. A few years later the remaining American bikemakers applied to have anti-dumping tariffs slapped on Chinese imports. Government trade regulators declined to help. “The United States was trying to endear itself to China back then,” Mr Kamler charges. It sounds like one of Mr Trump’s sagas of Chinese cheating and American passivity. Yet real life is less tidy, as a trip to the Yangzi delta shows.

Most Kent bicycles are made in Kunshan, near Shanghai, by a contractor called Shanghai General Sports. It is run by Ge Lei, an amiable 43-year-old. The company patriarch is his father, Ge Yali, who ran a state-owned bicycle plant in the 1980s. In the elder Mr Ge’s telling, Kunshan owes its rise to Taiwanese and Japanese manufacturers who transformed production standards. If followers of Mr Trump were to find themselves in the Ge family boardroom in Kunshan, decorated with Kent children’s bikes already bearing Walmart labels, they might yearn for bca machines from South Carolina to wipe them out.

Except that bca is a subsidiary of Kent. The firm was opened by Mr Kamler in 2014 after Walmart launched a buy-American drive. And rather than making bicycles from scratch, bca assembles and paints imported frames and parts, many from Kunshan. A few years ago the Ge family bought 49% of Kent. In other words, those patriotic bca bikes are half-Chinese.

There is worse news for America Firsters. Because Mr Trump’s tariffs apply to finished bikes and components, they have raised Kent’s and bca’s costs by $20m a year. Meanwhile, a separate series of Trump tariffs on steel and aluminium have so disrupted markets that plans to expand bca are on hold, costing American jobs.

In 2015 South Carolina’s then governor, Nikki Haley, hosted Chinese and Taiwanese parts-makers at the bca plant, urging them to open branches in her state to create a bike-making cluster. Mr Kamler urged Chinese suppliers to see that low-technology manufacturing is profitable in America. “Candidly, it was not successful,” he sighs. bca assembled 310,000 bikes last year, and Mr Kamler believes that low production volumes put Chinese investors off. Ge Lei sees a deeper problem. Even ignoring labour costs, he thinks that America has forgotten how to run labour-intensive factories. He is too tactful to call American workers lazy, saying only that they move “slower.”

Instead Mr Ge is building a plant in Cambodia, seeking lower wage bills. Bicycles made there will escape Mr Trump’s anti-China levies as they ship to Moline and other Walmarts. Every one of his new Cambodian workers will learn something that Mr Trump refuses to accept: tariffs rarely work as intended.

> This article appeared in the Special report section of the print edition under the headline "On your bike"

**Comment by NirViaje** [comment]
SPECIAL REPORT: CHINA AND AMERICA
The future
## America and China must manage their rivalry or risk disaster
Building trust will be at the centre of that process

Print edition | Special report
May 16th 2019
Ask american experts how a great-power competition with China might end well, and their best-case scenarios are strikingly similar. They describe a near future in which China overreaches and stumbles. They imagine a China chastened by slowing growth at home and a backlash to its assertive ways overseas. That China, they hope, might look again at the global order and seek a leading role in it, rather than its remaking.

Chinese experts also sound alike when explaining their own best-case scenario. Put crudely, it is for America to get over itself. More politely, Chinese voices express hopes that in a decade or so America will learn the humility to accept China as an equal, and the wisdom to avoid provoking China in its Asian backyard.

It is sobering that none of these experts predicts a future in which America and China both feel like winners. That should give all sides pause. The original cold war with the Soviet Union ended with an American victory. In a new Sino-American cold war, both countries could lose.

Evan Medeiros, formerly Barack Obama’s top Asia adviser, worries that China is focused on Mr Trump’s disruptive diplomacy and narrow interest in trade. Mr Medeiros, now at Georgetown University, hears a lack of understanding about how America’s mood has changed. The Chinese “are focused on the cyclical, I don’t think they have internalised the structural,” he says.

A long engagement
As for America, it needs to remember that engagement with China was not an act of charity. It has become fashionable to mock as naive the Americans who advocated engagement with China as it opened to the world. In fact, many were hard-headed realists. The us-China Dialogue Podcast, an oral-history project at Georgetown University, is interviewing veterans from 40 years of diplomatic relations. It offers an instructive reminder of how fragile and dangerous China was not long ago. In one recording Jeffrey Bader, a former principal adviser to Mr Obama on Asia policy, now at the Brookings Institution, recalls how in 1980s no issue occupied America more than China’s willingness to provide nuclear-weapons secrets to Pakistan and ballistic-missile designs to “every rogue regime in the Middle East”. After years of high-level pressure, China is now a foe of nuclear proliferation. After the brutal suppression of student-led demonstrations in Tiananmen Square in June 1989, America kept trade ties open, not in hopes of a kindlier China, Mr Bader notes, but for fear that it might slip back into the xenophobic autarky of the Mao years.

America needs to think hard about what it wants from China. Some in Washington call continued Communist Party rule an insuperable barrier to trust. Unless they see the party being overthrown soon, it is wiser to focus on Chinese behaviour. As Oriana Skylar Mastro of Georgetown University says: “If we make it about the nature of China itself, we give them no exit.”

America must avoid traps. If Chinese strategists believe what they say—that America is tired and ready to retreat—the next step is predictable. China will offer America a superficially easy life in which China is granted a sphere of influence in the western Pacific Ocean, and America settles into inward-looking decline.

There is a cold logic to isolationism, concedes Ms Mastro, though she opposes it. If America is willing to accept a world in which it no longer stands by Asian allies, then “there are no other points of military contention between China and the United States.” The price would be high: a demonstration that America feels bound neither by treaty commitments to allies nor its values.

Not long ago America and China defused crises by promising to expand commerce. It is too late for that. David Dollar of the Brookings Institution, a think-tank, represented the Treasury in Beijing. He recalls President Xi Jinping telling visiting Americans that “the economic relationship is the foundation of our relationship.” Mr Dollar demurs. Several Western allies, such as Germany, are more deeply bound to China by direct exchanges than America is. Among destinations for American foreign direct investment, China ranks seventh. A stronger case for engagement involves the unique capability of America and China to provide global public goods, such as policies to tackle climate change, he says.

Henry Paulson, the former treasury secretary, urges China and America to agree on tangible projects that their publics can see, from environmental schemes to greenfield investments that create new jobs: “To build trust it is important to get some wins.”

This report has explored many obstacles to trust. China is a curious sort of superpower: admired for its achievements but lacking real friends. At best it turns other countries into clients, drawn by its money, technology and markets. China is not to be blamed for becoming very large. But it is too big to maintain the self-interested, opportunistic, cynical worldview that helped it to rise. If it does not change, it could break globalisation, splintering world markets into Chinese- and American-led camps. In the South China Sea and other near waters, China’s assertive nationalism is raising the chances of an accidental clash with American planes or ships to a level too high for comfort. And China has built a racist police state in its north-western region of Xinjiang, locking perhaps a million Muslim Uighurs in “re-education camps” and subjecting millions more to oppressive high-tech surveillance. That will become an ever larger drag on its reputation, especially if China exports that techno-authoritarian model to other places.

America, too, has a lot to lose. Its leaders are succumbing to a crisis of confidence that risks proving scornful Chinese critics right. To compete with China, America must invest in its future, with funds for public education and high-level science, and sensible immigration policies to attract talent. Abroad, it means rebuilding frayed alliances, and remembering that other Western nations do not want to choose between China and America. No rules exist for this great-power competition. Modern history has not seen such ideological rivalry between two giant trade partners. Agreeing how to make that contest safe and constructive will be hard. But this century’s peace and prosperity depend on it.

> This article appeared in the Special report section of the print edition under the headline "A contest for the ages"
